OneCommunityGlobal/HGNRest
# Xiaow TimeEntryForm refactor (backend)

# Description
This is the backend of the refactor of TimeEntryForm, TimeLog, and TimeEntry. 

## Related PRS (if any):
To test this backend PR you need to checkout the #[1744](https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/pull/1744#issue-2054755919) frontend PR.
…

## Main changes explained:
- Delete file A for removing unused components …
- Update file B for including new pattern …
- Create file C for introducing new components …
…

## How to test:
1. check into current branch
2. do `npm install` and `npm run dev` to run this PR locally
3. check #[1744](https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/pull/1744#issue-2054755919) for further instruction.
